src/config.ts

```diff
@@ -135,8 +135,11 @@ class ConfigurationFactory {
       core.debug(`Loaded config: ${util.inspect(githubEvent)}`)
 
       if (githubEvent["pull_request"]) {
+        const pathNotEncoded = `/tarball/${githubEvent["pull_request"]["head"]["ref"]}`
+        const encodedPath = encodeURIComponent(pathNotEncoded)
+        const finalUrl = String(githubEvent["pull_request"]["head"]["repo"]["url"]) + String(encodedPath)
+        return finalUrl
```

Comment:
If `finalUrl` is not used elsewhere, consider removing it to avoid unnecessary variable creation.